src/impl/win.cc

```diff
@@ -108,6 +108,20 @@ Serial::SerialImpl::reconfigurePort ()
     THROW (IOException, "Error getting the serial port state.");
   }
 
+  // Boilerplate default settings
+  dcbSerialParams.fBinary = TRUE;
+  dcbSerialParams.fDtrControl = DTR_CONTROL_ENABLE;
+  dcbSerialParams.fDsrSensitivity = FALSE;
+  dcbSerialParams.fTXContinueOnXoff = FALSE;
+  dcbSerialParams.fOutX = FALSE;
+  dcbSerialParams.fInX = FALSE;
+  dcbSerialParams.fErrorChar = FALSE;
+  dcbSerialParams.fNull = FALSE;
+  dcbSerialParams.fRtsControl = RTS_CONTROL_ENABLE;
+  dcbSerialParams.fAbortOnError = FALSE;
+  dcbSerialParams.fOutxCtsFlow = FALSE;
```

Comment:
Some of these settings are set lower in this function, so it seems this line will not have any effect. Can you only set the ones that are not set elsewhere? These are the ones I see that are redundant, but you should double check me :)

- `fOutX`
- `fInX`
- `fRtsControl`
- `fOutxCtsFlow`

Reply:
fwiw I was seeing the same thing with an Arduino uno and was able to get it working by adding only the following line:

```
dcbSerialParams.fDtrControl = DTR_CONTROL_ENABLE;
```